http.go

```
@@ -107,3 +109,17 @@ func parseServerName(host string) string {
 	// anyways serverName is returned
 	return serverName
 }
+
+// Extracts the real host name from the HTTP request,
+func extractRealHostname(req *http.Request) (string) {
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)